src/server/worker/pipeline/transform/common_test.go

```diff
@@ -118,6 +119,10 @@ func (td *testDriver) NewSQLTx(cb func(*sqlx.Tx) error) error {
 	return td.inner.NewSQLTx(cb)
 }
 
+func (td *testDriver) AddSpanToAnyPipelineTrace(operation string, kvs ...interface{}) (opentracing.Span, context.Context) {
+	return td.inner.AddSpanToAnyPipelineTrace(operation, kvs...)
```

Comment:
I would like to add a tag to identify the worker but am not aware of any worker IDs we store

Reply:
I believe we've used worker IP in the past. I'll have to look up how to get it but there's a recipe and we (used to?) use it.

Reply:
figured this one out: `env.Config().PPSWorkerIP`
https://github.com/pachyderm/pachyderm/blob/master/src/server/cmd/worker/main.go#L70